src/main/scala/components/control.scala

```diff
@@ -39,36 +40,39 @@ class Control extends Module {
 
   val signals =
     ListLookup(io.opcode,
-      /*default*/           List(false.B, false.B, 3.U,   false.B, false.B,  false.B, false.B,    0.U,    0.U),
-      Array(                 /*  branch,  memread, toreg, add,     memwrite, immediate, regwrite, alusrc1,  jump */
+      /*default*/           List(false.B, false.B, false.B, 4.U,   false.B, false.B,  false.B, false.B,    0.U,    0.U),
```

Comment:
What's up with all of these changes? I would have expected only a single line to change (and maybe an extra bit added to the others).

Reply:
Just to make sure again, this change should stay?